harambe/pagination.py

```diff
@@ -68,6 +68,14 @@ def on_paginate(self, next_url: str) -> bool:
         self.current_page += 1
         return False
 
+    def on_save_cookies(self, cookies: List[dict[str, Any]]) -> bool:
+        """
+        Save cookies and check if they are duplicated
+        :param cookies: List of cookie dictionaries
+        :return: bool indicating if the cookies are duplicated, true if they are duplicated
+        """
+        self._add_data(cookies)
+
```

Comment:
Its ok for this not to handle this right now

Reply:
do you need  the even to just `pass`
or don't notify the class DuplicateHandler observer at all?

Reply:
yeah don't think duplicate handler needs this